lmcache/storage_backend/abstract_backend.py

```diff
@@ -124,6 +124,22 @@ def batched_get(
             else:
                 yield None
 
+    def batched_contains(self, keys: Iterable[CacheEngineKey],
+                         total_size: float) -> Iterable[bool]:
+        """
+        Query if the keys are in the cache or not in a batched manner
+
+        :param keys: the iterator of keys of the token chunks, including prefix 
+                hash and format
+
+        :return: the iterator of boolean values, indicating if the key is in 
+                the cache or not
+        """
```

Comment:
What does `total_size` mean here? It's not in the docstring. Also, why querying the existence of the keys requires `total_size`?